cmd/ci-operator/main.go

```diff
@@ -625,7 +626,24 @@ func (o *options) resolveInputs(steps []api.Step) error {
 	// TODO: instead of mutating this here, we should pass the parts of graph execution that are resolved
 	// after the graph is created but before it is run down into the run step.
 	o.jobSpec.Namespace = o.namespace
-	log.Printf("Using namespace %s", o.namespace)
+
+	//If we can resolve the field, use it. If not, don't.
+	var consoleHost string
+	routeGetter, err := routeclientset.NewForConfig(o.clusterConfig)
+	if err != nil {
+		log.Printf("[WARN] could not get route client for cluster config: %v", err)
```

Comment:
Don't use formatters like `[WARN]` and don't spit errors into the developer's log.

Reply:
Removed.